addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -363,10 +363,13 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
             model: 'ir.model',
             method: 'get_compatible_form_models',
         });
-        
+
```

Comment:
@abla001 The commit you mention in your commit message is not a merged one:
![image](https://github.com/odoo/odoo/assets/10338094/95b4f9e9-0e05-4268-9ecf-8bef2da43045)

Could you take the merged one, always, otherwise it is annoying to find the PR, know by who it was merged, etc.